views/movies/movies.hbs

```diff
@@ -4,5 +4,7 @@
   <div>
     <h2>Title: {{title}}</h2>
 
+    <a href="/movies/movie-details">See details</a>
```

Comment:
Here you're passing the path to the `hbs` file in your folder structure, but when you have an `<a>` tag the link that you need to pass inside `href` is the actual link that will appear in your browser, it's the path in the first line of code of the route:

```
router.get("/movies/:id", async (req, res, next) => {
  try {

(...)
```

```
<a href="/movies/{{_id}}">See details</a>
```